cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureEvalType.java

```diff
@@ -0,0 +1,67 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
```

Comment:
Question:  Do we not generate copyright headers for our source files?

Reply:
we do not